samples/snippets/src/test/java/com/example/logging/LoggingIT.java

```diff
@@ -41,16 +43,38 @@
 public class LoggingIT {
 
   private static final String TEST_LOG = formatForTest("test-log");
-  private static final String GOOGLEAPIS_AUDIT_LOGNAME = "cloudaudit.googleapis.com%2Factivity";
   private static final String STRING_PAYLOAD = "Hello, world!";
   private static final String STRING_PAYLOAD2 = "Hello world again";
 
   private ByteArrayOutputStream bout;
   private PrintStream out;
-  private Logging logging = LoggingOptions.getDefaultInstance().getService();
+  private static Logging logging;
+
+  private void deleteLog(String logName) throws InterruptedException {
+    int deleteAttempts = 0;
+    int allowedDeleteAttempts = 5;
+    boolean deleted = false;
+    while (!deleted && deleteAttempts < allowedDeleteAttempts) {
+      deleted = logging.deleteLog(logName);
+      deleteAttempts++;
```

Comment:
nit: log deleting log attempts so when we see a failing test, it's easier to understand what happened.

Reply:
Lines 64-67 print to STDERR in a case the log is not deleted. Due to sequential nature of the tests, if after few fail attempts the log is deleted, the number of failed attempts is not important.